src/components/swap/PendingModalContent/index.tsx

```diff
@@ -146,70 +145,89 @@ function getContent(args: ContentArgs): PendingModalStep {
     chainId,
   } = args
 
-  switch (step) {
-    case ConfirmModalState.WRAPPING:
+  const pendingConfirmationContent = useMemo(() => {
+    const title = swapPending ? t`Swap submitted` : swapConfirmed ? t`Swap success!` : t`Confirm Swap`
+    const tradeSummary = trade ? <TradeSummary trade={trade} /> : null
+    if (swapPending && trade?.fillType === TradeFillType.UniswapX) {
       return {
+        title,
+        subtitle: tradeSummary,
+        bottomLabel: (
+          <ExternalLink href="https://support.uniswap.org/hc/en-us/articles/17515415311501" color="textSecondary">
+            <Trans>Learn more about swapping with UniswapX</Trans>
+          </ExternalLink>
+        ),
+      }
+    } else if ((swapPending || swapConfirmed) && chainId && swapResult && swapResult.type === TradeFillType.Classic) {
+      const explorerLink = (
+        <ExternalLink
+          href={getExplorerLink(chainId, swapResult.response.hash, ExplorerDataType.TRANSACTION)}
+          color="textSecondary"
+        >
+          <Trans>View on Explorer</Trans>
+        </ExternalLink>
+      )
+      const showSubmittedState = swapPending && chainId === ChainId.MAINNET
+      if (swapPending) {
+        return {
+          title,
+          subtitle: showSubmittedState ? explorerLink : tradeSummary,
+          bottomLabel: showSubmittedState ? t`Transaction pending...` : explorerLink,
+        }
+      } else {
+        return {
+          title,
+          subtitle: explorerLink,
+          bottomLabel: null,
+        }
+      }
```

Comment:
these lines implement the change to remove the `TradeSummary` and replace it with the explorer link. all other changes in this file are just moving code around to make this function a hook